torch/_C/_distributed_c10d.pyi

```diff
@@ -653,14 +653,17 @@ class _SymmetricMemory:
         stride: torch.types._size,
         dtype: torch.dtype,
         device: torch.device,
-        group_name: str,
+        group_name: str | None = None,
```

Comment:
Do we really need to keep backwards compatibility? SymmetricMemory is a prototype feature, right?

Then again, I'm probably chill with this idea only because I haven't yet landed my code that makes use of SymmetricMemory :P

Reply:
> Do we really need to keep backwards compatibility?

No

> SymmetricMemory is a prototype feature, right?

You are right. And this function is a private pybind of a prototype feature. I just wanted make the change less disruptive to early adopters :)